python/xorq/cli.py

```diff
@@ -8,54 +8,50 @@
 from xorq.vendor.ibis import Expr
 
 
-def build_command(script_path, expression, builds_dir="builds"):
+def build_command(script_path, expr_name, builds_dir="builds"):
     """
     Generate artifacts from an expression in a given Python script
 
     Parameters
     ----------
     script_path : Path to the Python script
-    expression : The name of the expression to build
+    expr_name : The name of the expression to build
     builds_dir : Directory where artifacts will be generated
 
     Returns
     -------
 
     """
 
-    if len(expression) != 1:
-        print("Expected one, and only one expression", file=sys.stderr)
-        sys.exit(1)
-
-    (expression,) = expression
-
     if not os.path.exists(script_path):
         print(f"Error: Script not found at {script_path}", file=sys.stderr)
         sys.exit(1)
 
-    print(f"Building {expression} from {script_path}")
+    print(f"Building {expr_name} from {script_path}")
 
     build_manager = BuildManager(builds_dir)
 
     vars_module = import_from_path(script_path)
 
-    if not hasattr(vars_module, expression):
-        print(f"Expression {expression} not found", file=sys.stderr)
+    if not hasattr(vars_module, expr_name):
+        print(f"Expression {expr_name} not found", file=sys.stderr)
         sys.exit(1)
 
-    expr = getattr(vars_module, expression)
+    expr = getattr(vars_module, expr_name)
 
     if not isinstance(expr, Expr):
         print(
-            f"The object {expression} must be an instance of {type(expr)}",
+            f"The object {expr_name} must be an instance of {type(expr)}",
             file=sys.stderr,
         )
         sys.exit(1)
 
     expr_hash = build_manager.compile_expr(expr)
     print(
-        f"Written '{expression}' to {build_manager.artifact_store.get_path(expr_hash)}"
+        f"Written '{expr_name}' to {build_manager.artifact_store.get_path(expr_hash)}",
+        file=sys.stderr,
     )
+    print(build_manager.artifact_store.get_path(expr_hash))
```

Comment:
Why are we printing this?

Reply:
per the commit message
> so scripts can use stdout to compose operations

Reply:
But we are also printing the previous line, should we print both?